tests/puppeteer/campaigns/data/faker/invoice.js

```diff
@@ -2,7 +2,7 @@ const faker = require('faker');
 
 module.exports = class Invoice {
   constructor(invoiceOptions = {}) {
-    this.invoiceNumber = invoiceOptions.invoiceNumber || faker.random.number({min: 1, max: 200}).toString();
+    this.invoiceNumber = invoiceOptions.invoiceNumber || faker.random.number({min: 20, max: 200}).toString();
```

Comment:
@nesrineabdmouleh Why do you change it to 20 ?

Reply:
There is an error when we put a number lower than the last invoice number, in our case, there are 8 invoice creations before this test, then we should enter a value upper than 9, I chose 20 because maybe we will add more tests in the future :)

Reply:
Why not directly doing this?
```suggestion
    this.invoiceNumber = invoiceOptions.invoiceNumber || faker.random.number({min: 100, max: 200}).toString();
```